stable/yugabyte/templates/service.yaml

```diff
@@ -218,10 +218,19 @@ spec:
           postStart:
             exec:
               command:
-                - "sh"
+                - "bash"
                 - "-c"
                 - >
                   mkdir -p /mnt/disk0/cores;
+                  pattern="$(sysctl kernel.core_pattern --values)";
+                  core_dir="${pattern%/*}";
+                  if [ "${core_dir:0:1}" = "/" ]; then
+                    if [ ! -d "${core_dir}" ]; then
+                      # create the parent directory
+                      mkdir -p "${core_dir%/*}";
+                      ln -s "/mnt/disk0/cores" "${core_dir}";
```

Comment:
we should make sure none of these can cause the postStart to fail, because that would cause the container to never reach running state. Since this step is non-critical, I would add a `|| true` to it

Reply:
Yep, made the changes.